extension/src/commands/util.ts

```diff
@@ -18,3 +20,40 @@ export const showSetupOrExecuteCommand =
 
     return callback(context)
   }
+
+const enum PLOT_TYPE {
+  CUSTOM = 'custom',
+  TOP_LEVEL = 'top-level'
+}
+
+export const addPlotCommand = async (context: Context) => {
```

Comment:
[I] These should go in a new `plots/quickPick` file. Can you also please check to see if we will get two analytics events each time this is called (I don't think we want that).

Reply:
> Can you also please check to see if we will get two analytics events each time this is called (I don't think we want that).

Oh, I think we would get two since `addPlot` fires one and then "add custom"/"add pipeline" would fire another. We could remove the "add custom" and "add pipeline" commands and add a `type` argument to the "add plot" command 🤔